### DISCUSSION TITLE
How can I learn algorithms faster instead of just leetcode?

### DISCUSSION THREAD

**Discussion by Angellin0114** [bluetch/leetcode-javascript#6] [Ideas]
How can I learn algorithms faster instead of just leetcode?

**Comment by bluetch** [comment]
Watch some programming videos such as Udemy and YouTube, follow the directions, some of speakers raise great example would help you understand better.

**Comment by bluetch** [comment]
such as this one: https://www.udemy.com/course/algorithm-data-structure/

**Comment by Angellin0114** [comment]
thanks
